documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -143,21 +143,21 @@ You configure Hibernate OGM and Infinispan in two steps basically:
 * If you need to run JPQL or HQL queries, add Hibernate Search on the classpath
   (<<ogm-query-using-hibernate-search>>)
 
-Note that, except when using [acronym]`JNDI`, Hibernate OGM will bootstrap and Infinispan Embedded node
-in your same JVM, and terminate it on shutdown of the Hibernate instance.
+Note that, except when using [acronym]`JNDI`, Hibernate OGM will bootstrap Infinispan Embedded 
+in the same JVM and terminate it on shutdown of the Hibernate OGM factory.
 
 [CAUTION]
 ====
-If you have Hibernate OGM boot Infinispan using a clustered configuration, this might automatically join the cluster of
-other Infinispan nodes running in your network: the default is automatic discovery!
+Hibernate OGM will start Infinispan Embedded in automatic discovery.
+If you are using a clustered configuration might automatically join existing Infinispan clusters
```

Comment:
What do think instead of `...configuration, it might automatically join an existing Infinispan cluster`?

Reply:
ok